sideloader/postinstall.sh

```diff
@@ -0,0 +1,9 @@
+pip="${VENV}/bin/pip"
+
+cd "${INSTALLDIR}/${REPO}/"
+
+$pip install -e "${INSTALLDIR}/${REPO}/"
+
+springboard="${VENV}/bin/springboard"
+
+$springboard bootstrap
```

Comment:
no, I live it in there and just do a deployment again after the file was created. It's likely the first deployment will fail since puppet will not have run yet.